### PR TITLE
scripts review, create-group, user-to-group

### PR DESCRIPTION
**Pull request by avoidik**

### CODE REVIEW

File: gerrit-user-to-group.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+fi
+
+# Add user to group
+echo "Adding user: ${username}, to group: ${target_group}"
+count=0
+until [ $count -ge ${MAX_RETRY} ]
+do
+  json_request="{ \"members\": [ \"${username}\" ] }"
+  ret=$(curl --request POST --user "${admin_user}:${admin_password}" --header 'Content-Type: application/json; charset=UTF-8' --data "${json_request}" --output /dev/null --silent --write-out "%{http_code}" http://localhost:8080/gerrit/a/groups/"${target_group}"/members.add)
+  if [[ ${ret} -eq 201 ]]; then
```
Comment (thinkernel) [inline gerrit-user-to-group.sh]: The return code should be 200 here.

**Comment by thinkernel** [comment]
This patch looks good except a little mistake which I've already commented on it.

**Comment by avoidik** [comment]
okay, rebasing

**Comment by thinkernel** [comment]
Hi @avoidik 
Your patches have been merged. Thank you very much.
However I found an issue #4 which is not caused by your patch but will prevent this container works properly on the first run. I have to restart the container to make it works again. I wonder if the same thing happens in your box?

**Pull request merged**